genjson.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/bash
```

Comment:
This file is here for some reason or is it just WIP?

Reply:
WIP. But I guess I don't have to keep it in git.